yt_dlp/extractor/videocampus_sachsen.py

```diff
@@ -162,7 +181,86 @@ def _real_extract(self, url):
             'id': video_id,
             'title': title,
             'description': description,
+            'thumbnail': thumbnail,
             'display_id': display_id,
             'formats': formats,
-            'subtitles': subtitles
+            'subtitles': subtitles,
         }
+
+
+class ViMPPlaylistIE(InfoExtractor):
+    IE_NAME = 'ViMP:Playlist'
+    _VALID_URL = r'''(?x)(?P<host>https?://(?:%s))/(?:
+        album/view/aid/(?P<album_id>[0-9]+)|(?P<c_mode>category|channel)/(?P<c_name>[\w-]+)/(?P<c_id>[0-9]+)
+        )''' % ('|'.join(map(re.escape, VideocampusSachsenIE._INSTANCES)))
```

Comment:
a bit hard to read
```suggestion
    _VALID_URL = r'''(?x)(?P<host>https?://(?:%s))/(?:
        album/view/aid/(?P<album_id>[0-9]+)|
        (?P<mode>category|channel)/(?P<name>[\w-]+)/(?P<id>[0-9]+)
    )''' % '|'.join(map(re.escape, VideocampusSachsenIE._INSTANCES))
```
I also changed the group names. Change below accordingly